trust-center-agent/src/main/java/care/smith/fts/tca/consent/configuration/GicsConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package care.smith.fts.tca.consent.configuration;
+
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Slf4j
+@Configuration
+@ConfigurationProperties(prefix = "consent.gics")
+@Data
+public class GicsConfiguration {
+  private Integer pageSize = null;
+
+  GicsFhirConfiguration fhir;
+
+  @Bean
```

Comment:
This looks strange, an unnamed bean of type int. We should qualify it explicitly, not only by method name.